### DISCUSSION TITLE
Orange Pi 5 Plus HDMI INPUT '

### DISCUSSION THREAD

**Discussion by radekolszak** [Joshua-Riek/ubuntu-rockchip#772] [General]
After making "ls /dev/"  command on OrangePi5Plus there is no '/dev/video0' . I have only video-dec0 and video-enc0
Joshua-Riek Ubuntu 24.04 LTS desktop.
I want to make hdmi input in OBS using this tutorial. 
https://www.youtube.com/watch?v=i_L0mGepOtI

**Comment by Joshua-Riek** [comment]
You need to enable the hdmirx overlay, this is not enabled by default due to https://github.com/Joshua-Riek/ubuntu-rockchip/issues/606. 

Please see the wiki for how to set a device tree overlay.
https://github.com/Joshua-Riek/ubuntu-rockchip/issues/606
